library/src/main/kotlin/com/chuckerteam/chucker/internal/ui/transaction/TransactionAdapter.kt

```diff
@@ -120,14 +120,10 @@ internal class TransactionAdapter internal constructor(
     }
 }
 
-private fun ChuckerListItemTransactionBinding.displayGraphQlFields(graphQlOperationName: String?) {
-    if (graphQlOperationName != null) {
+private fun ChuckerListItemTransactionBinding.displayGraphQlFields(transaction: HttpTransactionTuple) {
+    if (transaction.graphQlOperationName != null || transaction.isGraphQLRequest) {
         graphqlIcon.visibility = View.VISIBLE
-        graphqlPath.visibility = View.VISIBLE
-
-        graphqlPath.text = graphQlOperationName
```

Comment:
Why are you removing this information? I would rather display `<Unable to discover GraphQL operation name>` if operation name is missing

Reply:
My apologies. I'd not thought of displaying a text in the event the graphql operation is missing.